tests/utils/shopper.js

```diff
@@ -47,6 +47,71 @@ export const shopper = {
 		);
 	},
 
+	placeOrder: async () => {
+		await expect( page ).toClick( '.wc-block-components-button__text', {
+			text: 'Place Order',
+		} );
+		page.waitForNavigation( {
```

Comment:
await is missing here :)

Reply:
```
await page.waitForNavigation( {
			waitUntil: 'networkidle0',
		} );
```

Reply:
```suggestion
		await page.waitForNavigation( {
```

Reply:
actually let's do it like [WC does it](https://github.com/woocommerce/woocommerce-monorepo/blob/27009cdc0a2af1cca88b88b5ee3600416bab6b7a/plugins/woocommerce/tests/e2e/utils/src/flows/shopper.js#L71-L76), using `await Promise.all`

```
	placeOrder: async () => {
		await Promise.all( [
			await expect( page ).toClick( '.wc-block-components-button__text', {
			text: 'Place Order',
		} ),
			page.waitForNavigation( { waitUntil: 'networkidle0' } ),
		] );
	},
```